java-manta-client-unshaded/src/main/java/com/joyent/manta/client/MantaClient.java

```diff
@@ -720,51 +720,120 @@ public MantaDirectoryListingIterator streamingIterator(final String path, final
             }
         }
 
```

Comment:
I noticed we were missing these additional characteristics of the stream. I went ahead and added them.